.github/workflows/continuous-integration-arm64.yml

```
@@ -0,0 +1,50 @@
+name: github-arm64
+
+on:
+  workflow_call:
+
+concurrency:
+  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}-arm64
+  cancel-in-progress: ${{github.event_name == 'pull_request'}}
+
+permissions: read-all
+
+jobs:
+  arm64ci:
+    name: arm64-ci
+    runs-on: [ubuntu-24.04-arm]
+    container:
+      image: ghcr.io/kokkos/ci-containers/ubuntu:latest-arm64
+
+    strategy:
+      matrix:
+        include:
+          - backend: "SERIAL"
+            cmake_build_type: "RelWithDebInfo"
+          - backend: "THREADS"
+            cmake_build_type: "RelWithDebInfo"
+          - backend: "SERIAL"
+            cmake_build_type: "Debug"
+          - backend: "SERIAL"
+            cmake_build_type: "Release"
```

```suggestion
          - backend: "OPENMP"
            cmake_build_type: "RelWithDebInfo"
          - backend: "THREADS"
            cmake_build_type: "RelWithDebInfo"
          - backend: "SERIAL"
            cmake_build_type: "Release"
```

Thinking more about this, I don't see think the proposed set is particular good spread for this. Does the compiler on this support OpenMP? If so I think that is more relevant. The Debug also doesn't matter specifically for ARM. What ARM testing is good for is to find memory order bugs due to implicit relying on x86 strong ordering. So the threaded backends are really primary concern, in particular in release mode.